db/controllers/article.controllers.js

```diff
@@ -1,4 +1,12 @@
-const {retrieveArticleById} = require("./../models/article.models")
+const {retrieveArticleById, retrieveAllArticles,} = require("./../models/article.models")
+
+exports.getArticles = (req, res, next) => {
+    retrieveAllArticles()
+    .then((article) => {
+        res.status(200).send({ article: article })
```

Comment:
At the risk of being picky, you're returning a lot of articles, the key should reflect that.